src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -204,10 +204,8 @@ class Loader {
 
     // Use this name for the module wrapper for consistency with node.
     const evalResultVariable = 'Object.<anonymous>';
```

Comment:
Pretty sure that depends on where you throw (during init vs async), will check if I can dig up my repro case.